### PR TITLE
fix(UVE): Hide add forms option for Non-Licensed Users in UVE Editor 

### LINKED ISSUE #28900: Hide add forms option for Non-Licensed Users in UVE Editor
### Problem Statement

In the UVE editor of our content management system, the option to add forms is visible to all users, regardless of their license status. Since forms are an Enterprise Edition (EE) feature, users without a proper license should not see this option. This discrepancy can lead to confusion and attempts to use features that are not available to them.

**FYI:** This is not happening on the regular editor 

### Steps to Reproduce

- Log into the content management system
- Make sure you have enabled the UVE editor
- Edit a page 
- Observe that the option to add forms is available despite the lack of an appropriate license.

### Acceptance Criteria

**Hide Add Forms Option:** Ensure that the option to add forms is hidden in the UVE editor for users who do not have an Enterprise Edition license as we do in the regular editor.

### dotCMS Version

Tested on the latest trunk // Docker // FF 

### Proposed Objective

Quality Assurance

### Proposed Priority

Priority 2 - Important

### PR DESCRIPTION
**Pull request by rjvelazco**
### Proposed Changes
* Hide add forms option for Non-Licensed Users in UVE Editor

### Video

https://github.com/dotCMS/core/assets/72418962/631e2dd0-be2c-4034-9098-890ac4f1eae8

This PR fixes: #28900

### CODE REVIEW

**Review by fmontes — APPROVED** [review]

**Review by KevinDavilaDotCMS — APPROVED** [review]

File: core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.ts [hunk 1]
```diff
@@ -41,20 +42,22 @@ export class EmaContentletToolsComponent implements OnChanges {
...
     @Output() addContent = new EventEmitter<ActionPayload>();
     @Output() addForm = new EventEmitter<ActionPayload>();
     @Output() addWidget = new EventEmitter<ActionPayload>();
     @Output() edit = new EventEmitter<ActionPayload>();
     @Output() editVTL = new EventEmitter<VTLFile>();
     @Output() delete = new EventEmitter<ActionPayload>();
 
-    items: MenuItem[] = [
+    private dotMessageService = inject(DotMessageService);
+    private buttonPosition: 'after' | 'before' = 'after';
```
Comment (nicobytes) [inline core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.ts:59]: @rjvelazco According to recent discussions, we should use native private attributes by ES6 with `#`.

File: core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.spec.ts [hunk 2]
```diff
@@ -140,14 +141,10 @@ describe('EmaContentletToolsComponent', () => {
...
-                it('should call addForm on Form option click', () => {
-                    const addSpy = jest.spyOn(spectator.component.addForm, 'emit');
-                    spectator.click('[data-testId="add-top-button"]');
-                    spectator.click(byText('Form'));
-                    expect(addSpy).toHaveBeenCalledWith({
-                        ...contentletAreaMock.payload,
-                        position: 'before'
-                    } as ActionPayload);
+                it('should not call addForm on Form option click', () => {
+                    spectator.click('[data-testId="add-bottom-button"]');
```
Comment (nicobytes) [inline core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.spec.ts:145]: @rjvelazco When we use the data-testId pattern, I suggest using the `byTestId` selector from Spectator as a standard, like this:

```ts
import { byTestId } from '@ngneat/spectator';

spectator.click(byTestId('add-bottom-button'));
```

This way we avoid typos.

**Review by nicobytes — CHANGES_REQUESTED** [review]

File: core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.ts (hunk 1 above)
Comment (rjvelazco) [reply core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.ts:59]: Right!

File: core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.spec.ts (hunk 2 above)
Comment (rjvelazco) [reply core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/ema-contentlet-tools/ema-contentlet-tools.component.spec.ts]: Done!

**Review by nicobytes — APPROVED** [review]
LGTM

**Review by zJaaal — APPROVED** [review]

**Review by oidacra — APPROVED** [review]

**Pull request merged**